README.md

```diff
@@ -773,8 +773,6 @@ List the users who have a certain relation to a particular type.
 options := ClientListRelationsOptions{
     // You can rely on the model id set in the configuration or override it for this specific request
     AuthorizationModelId: openfga.PtrString("01GAHCE4YVKPQEKZQHT2R89MQV"),
-    // Max number of requests to issue in parallel, defaults to 10
```

Comment:
It looks like the change that added this hadn't made it across to sdk-generator. Created openfga/sdk-generator/pull/365

Reply:
Great catch! I'll pull those changes into my generator change and update this PR after running the gen again.

Reply:
I think we're good now, but please check ;). I made https://github.com/openfga/sdk-generator/pull/367 to patch back some of the README changes that were missed.

Reply:
Oops, thanks for catching that!